containers/src/Kokkos_UnorderedMap.hpp

```diff
@@ -186,6 +217,11 @@ class UnorderedMap {
  public:
   //! \name Public types and constants
   //@{
+  using default_op_type = typename UnorderedMapInsertOpTypes<
```

Comment:
Since you are always passing in `m_values` why not use the type of `m_values` here -- `value_type_view`